dissect/target/plugins/os/unix/linux/fortios/_os.py

```diff
@@ -2,56 +2,97 @@
 
 import gzip
 import io
+from base64 import b64decode
 from binascii import crc32
+from datetime import datetime
+from tarfile import ReadError
 from typing import BinaryIO, Iterator, Optional, TextIO
 
-from dissect.util import cpio
+from Crypto.Cipher import AES
+from dissect.util import cpio, ts
 from dissect.util.stream import OverlayStream
 
 from dissect.target.filesystem import Filesystem
 from dissect.target.filesystems.tar import TarFilesystem
-from dissect.target.helpers.record import UnixUserRecord
+from dissect.target.helpers.fsutil import open_decompress
+from dissect.target.helpers.record import TargetRecordDescriptor
 from dissect.target.plugin import OperatingSystem, export
 from dissect.target.plugins.os.unix.linux._os import LinuxPlugin
 from dissect.target.target import Target
 
+FortiOSUserRecord = TargetRecordDescriptor(
+    "fortios/user",
+    [
+        ("string", "name"),
+        ("string", "groups"),
+        ("string", "password"),
```

Comment:
You can make a field a list by appending `[]` to the type. E.g. `("string[]", "groups")`. As I see you doing some joins for these fields, this might be useful.

Reply:
Added in [e765ee2](https://github.com/fox-it/dissect.target/pull/509/commits/e765ee29af0ff2bb8f524c6c5eb01eb7932f9ff6).